articles/custom_storage_location.md

```diff
@@ -127,6 +127,41 @@ In **Permissions**, click **CORS configuration**. In the CORS configuration edit
 <br/>
 For more information, please read: [How Do I Configure CORS on My Bucket?](https://docs.aws.amazon.com/AmazonS3/latest/dev/cors.html#how-do-i-enable-cors){:target="_blank"}
 
+### Setup bucket with AWS cloudformation
+
+You can also use AWS Cloudformation to provision a custom Synapse bucket.
```

Comment:
why would someone want to do this?

Who is paying for it - Sage, or end user? Who is in control of it?

Can the bucket be used for anything other than Synapse?

Reply:
Not sure if these are suggestions for more docs or your asking because you don't know but to answer:
> why would someone want to do this?
As noted in the commit message this is an alternative to creating the bucket manually.  if someone wants to run a single command to quickly and easily get a synapse bucket then they can use cloudformation which we conveniently already have a script for this.

> who is paying for it?
The end user, or the owner of the account that the bucket gets provisioned to.

> can the bucket be used for anything other than synapse
yes, it's just a generic AWS S3 bucket.  The only difference is that synapse has access to the bucket.

Reply:
> As noted in the commit message this is an alternative to creating the bucket manually.

I think this should incorporate into the previous part of this documentation, explaining to users that there 2 ways to setup a bucket, and highlight the manual instructions with the same heading as the heading we use for using the CloudFormation script part.  

These answers are valuable to users as well. I had the same questions. So I think we should clearly address these.

With these understanding, I would immediately jump to the CloudFormation script and skip the manual instructions. Instead, with the current structure, I may follow all the manual instructions before reading the CF part. The heading does not make sense now, because this section has the same heading as a part of the manual instructions.